packages/round-manager/src/features/round/ViewApplicationPage.tsx

```diff
@@ -14,21 +11,41 @@ import { ReactComponent as TwitterIcon } from "../../assets/twitter-logo.svg"
 import { ReactComponent as GithubIcon } from "../../assets/github-logo.svg"
 import Footer from "../common/Footer"
 import { datadogLogs } from "@datadog/browser-logs"
-
+import { PassportVerifier } from "@gitcoinco/passport-sdk-verifier"
+import { ProjectCredentials } from "../api/types"
 
 type ApplicationStatus = "APPROVED" | "REJECTED"
 
+enum VerifiedCredentialState {
+  VALID,
+  INVALID,
+  PENDING
+}
+
+enum ApplicationQuestions {
+  GITHUB = "Github",
+  GITHUB_ORGANIZATION = "Github Organization",
+  TWITTER = "Twitter"
+}
+
+export const IAM_SERVER = "did:key:z6MkghvGHLobLEdj1bgRLhS4LPGJAvbMA1tn2zcRyqmYU5LC"
```

Comment:
should this be an environment variable?

Reply:
David and I were debating on whether to make this a constant or an env variable, but we thought a constant in the code would be fine:
- the value does not need to be a secret (it's the public key of the IAM server)
- the value will probably not change often (IAM server shouldn't change keys unless something drastic happens)

@chibie @thelostone-mc What do you think?